### ISSUE TITLE
Blocked elements via CSS selector should be shown on brave://adblock or on a custom page

### ISSUE THREAD

**Issue by srirambv** [brave/brave-browser#1280]
## Description 

Blocked elements via CSS selector should be shown on chrome://adblock

## Steps to Reproduce

   1. Visit any site and block an element via CSS selector on context menu
   2. If your memory serves you well, _good for you_, if not you'll forget what you have blocked on the site
   3. Someone else reports that ads/page elements are not blocked for the same site and you start to wonder why others can see but you can't 

## Actual result:

Blocked elements via CSS selector is not listed anywhere making it hard to remember what is blocked on a site

## Expected result:
Would be nice to list the items blocked on site under `chrome://adblock`

## Reproduces how often: 

Easy

## Brave version (chrome://version info)

All

### Reproducible on current release:

Doesn't apply since its block option is not there

### Website problems only:  
- Does the issue resolve itself when disabling Brave Shields? 
- Is the issue reproducible on the latest version of Chrome?

**Comment by Brave-Matt** [comment]
+1million for this. This feature has a lot of potential if it gets fleshed out.

**Comment by tomlowenthal** [comment]
Sounds like we need to implement some front-end UI for this functionality?

**Comment by Brave-Matt** [comment]
@tomlowenthal agreed!
Would also be cool if, after selecting it from the context menu, the pointer highlighted/outlined the element it's looking at. Kind of like the inspector. 

Would help people block what they want without having to know or dig through any css.

**Comment by srirambv** [comment]
+1 on the highlighting the element to be blocked.

**Comment by Brave-Matt** [comment]
From Community:
![image](https://user-images.githubusercontent.com/3299098/48603524-8fc94d00-e92b-11e8-91e6-d1e0cb59f04b.png)

**Comment by igneosaur** [comment]
Blocking elements doesn't even work for me at the moment. Ads are back when you refresh, CSS classnames don't change or anything.

**Comment by egid** [comment]
I'm seeing the exact same problem @igneosaur is reporting. When blocking, the inspector shows 'injected styles'; on reload, they are not applied.

**Comment by eljuno** [comment]
+1s from Community https://community.brave.com/t/centralized-css-element-blocking-management/54753?u=eljuno

**Comment by bernieke** [comment]
For me the block only lasts a while. Every few days or so I need to re-apply it for some reason.

Even just a simple txt page which shows the rules and the sites (or is it pages?) they are applied to would be extremely helpful! Or even just some commandline command to extract the same from the config somehow?

**Comment by bsclifton** [comment]
@bernieke that makes sense - we aren't persisting the rules to disk (right now, it's only stored in local storage). If your cookies get cleared, it'll delete the rules

A requirement for this task would be that the rules should be persisted to disk or at least in a similar manner to custom ABP rules

**Comment by veekas** [comment]
@bsclifton persistent storage would be great!

**Comment by bsclifton** [comment]
Closing as `wontfix` - we have a new cosmetic filter in place. You can put cosmetic filter rules on brave://adblock now 😄It's live in version 1.7 and newer. 

I created https://github.com/brave/brave-browser/issues/8914 to track removing this older version

**Comment by Cral-Cactus** [comment]
When you clear in brave://adblock custom css it still apply but just gone from textbox!
